src/domains/domains.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef DOMAINS_H_
+#define DOMAINS_H_
+
+#ifdef __cplusplus
+extern "C" {
```

Comment:
Could we make doamains.cpp as it is c++ application?

Reply:
Unfortunatelly not. It will require additional cpp files. Because files, which are including domain configuration are written on c style and can't be converted to cpp so easily.